constants/Enchants.json

```
@@ -258,6 +258,12 @@
       "goodLevel": 2,
       "maxLevel": 2
     },
+    "lapidary": {
+      "loreName": "Lapidary",
+      "nbtName": "lapidary",
+      "goodLevel": 1,
```

```suggestion
      "goodLevel": 0,
```
Tier 1 should already be treated as great to match the others (since it's not from the enchantment table).